### ISSUE TITLE
html images in markdown do not update when cell is rerun

### ISSUE THREAD

**Issue by SimonEnsemble** [jupyterlab/jupyterlab#4589]
If I put the following html tag in a markdown cell:

```
#### blah blah

<center>
    <img src="myimage.png" width=500>
</center>
```

and run the cell, the image displays properly. However, if I change the image `myimage.png`, go into the cell again and hit play again, the image displayed does not update; it instead displays the old image before it was updated. This is kind of a problem for when I'm preparing presentations in Jupyter Lab. Thanks.

**Comment by dansteingart** [comment]
I'm seeing a bug for all images in version `0.32.1` that did not exist in `0.31.12`

Here's 0.31.12

![image](https://user-images.githubusercontent.com/152047/40667746-dcc9fdf6-6330-11e8-985b-99449521b00e.png)

Here's 0.32.1

![image](https://user-images.githubusercontent.com/152047/40667795-fb010b02-6330-11e8-8c70-c8d65d27c7e5.png)

**Comment by ian-r-rose** [comment]
Hi @SimonEnsemble, sorry for the slow reply. Your image is not updating because we don't re-render the markdown cell if the source has not changed. We may want to revisit that decision, but for now you can get the altered image to show up by changing the text anywhere (by, for instance, adding a space somewhere).

@dansteingart I am unable to reproduce your issue with any images *except* the one you linked to. Might that file be corrupted in some way?

**Comment by dansteingart** [comment]
It works for any image I throw at it...
On Tue, May 29, 2018 at 11:45 AM, Ian Rose <notifications@github.com> wrote:
Hi @SimonEnsemble [https://github.com/SimonEnsemble] , sorry for the slow reply. Your image is not updating because we don't re-render the markdown cell if the source has not changed. We may want to revisit that decision, but for now you can get the altered image to show up by changing the text anywhere (by, for instance, adding a space somewhere).

@dansteingart [https://github.com/dansteingart] I am unable to reproduce your issue with any images except the one you linked to. Might that file be corrupted in some way?

**Comment by ian-r-rose** [comment]
Ah, I think this may be fixed on master (by #4613). On `0.32.1` we are mistakenly adding a query string to some URLs, which makes some external image loads succeed, and others fail. The the fix should be in the released packages in a week or two.

**Comment by jasongrout** [comment]
This issue is actually two separate issues. Closing as the original issue is answered, and the second issue should be fixed. @dansteingart - please comment below (or open another issue) if you are still not seeing an image.
